consensus/scheme/rolldpos/endorsementmanager.go

```diff
@@ -357,8 +346,8 @@ func (m *endorsementManager) AddVoteEndorsement(
 	}
 	m.collections[encoded] = c
 
-	if m.eManagerDB != nil && m.isMajorityFunc != nil {
```

Comment:
why `m.isMajorityFunc != nil ` could be deleted?

Reply:
Since endorsementmanager no longer keeps isMajorityFunc.